cfg/builder/builder_walk.cc

```diff
@@ -311,14 +311,14 @@ BasicBlock *CFGBuilder::walk(CFGContext cctx, ast::Expression *what, BasicBlock
 
                 if (s->block != nullptr) {
                     auto newRubyBlockId = ++cctx.inWhat.maxRubyBlockId;
-                    vector<ast::ParsedArg> blockArgs = ast::ArgParsing::parseArgs(cctx.ctx, s->block->args);
+                    vector<ast::ParsedArg> blockArgs = ast::ArgParsing::parseArgs(s->block->args);
                     vector<core::ArgInfo::ArgFlags> argFlags;
                     for (auto &e : blockArgs) {
                         auto &target = argFlags.emplace_back();
-                        target.isKeyword = e.keyword;
-                        target.isRepeated = e.repeated;
-                        target.isDefault = e.default_ != nullptr;
-                        target.isShadow = e.shadow;
+                        target.isKeyword = e.flags.isKeyword;
```

Comment:
I didn't know if I could do `target = e.flags` -- we're not copying over the `isBlock` flag here, and I was wondering if that is intentional? Or if it's just impossible so I should do `target = e.flags`

Reply:
>  I was wondering if that is intentional?

seems like a bug to me on first sight.

Let's fix it in a different PR?

Reply:
I'll open a PR once builds are fixed and I can land this. :)